spec/index.html

```diff
@@ -538,72 +538,70 @@ <h2>RDF Graphs</h2>
   <section id="section-triples">
     <h3>Triples</h3>
 
-    <p>An <dfn data-local-lt="triple">RDF triple</dfn> is a 3-tuple that is defined recursively as follows:</p>
+    <p>An <dfn data-local-lt="triple">RDF triple</dfn> (usually called "triple")
+      is a 3-tuple (|s|, |p|, |o|) where:</p>
+
+    <ul>
+      <li>|s| is an <a>IRI</a> or a <a>blank node</a>,</li>
```

Comment:
This document says above that a triple can have a triple term as a subject.

Reply:
@pfps

> This document says above that a triple can have a triple term as a subject.

I have [now changed](https://github.com/w3c/rdf-concepts/pull/78/files#r1494587309) that statement above in order to be aligned with the new definition here.

Reply:
I thought that triple terms were to be allowed as both subjects and objects.  Lots of use cases are best expresses with triple terms as subjects.

Reply:
See https://github.com/w3c/rdf-concepts/issues/80

Reply:
People are still being confused between Triple Terms and Occurrences (still needs a better name). A Triple Term only is intended to exist as the object of a triple where the predicate is `rdf:nameOf` (or similar). An occurrence in the abstract syntax is just another node (IRI or Blank Node to be determined).

Reply:
@TallTed your suggestion for this line is to add triple term as another option for what the subject may be. My response to this suggestion is in [my comment above](https://github.com/w3c/rdf-concepts/pull/78/files#r1494748017).

Reply:
Picking up in #80.